schema.json

```diff
@@ -612,7 +612,11 @@
         "properties": {
           "name": {
             "type": "string",
-            "description": "e.g. Philosophy"
+            "description": "e.g. Machine Learning"
+          },
+          "keywords": {
```

Comment:
Would still love to have a general notion, if the issue was understood correctly from @ss22ever 
On this specific code so. It would be one string for various keywords. We might make it a list/array of strings. That would make it easier to loop through etc. Also better data schema in my personal view. Happy to discuss this.

Reply:
@stp-ip  @dbriesz  as per #3 a general issue was raised needing discussion. So yes probably we should use list for this but then we should also make sure the number of items the keywords can hold so it does not populate the resume.